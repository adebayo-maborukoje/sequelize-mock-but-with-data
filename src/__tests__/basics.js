let Sequelize = require('sequelize');

let define = () => {
  let sequelize = new Sequelize();
  let User = sequelize.define('User', {
    name: Sequelize.STRING,
    email: Sequelize.STRING,
    age: Sequelize.INTEGER,
    verified: Sequelize.BOOLEAN,
  });
  return { sequelize, User };
}

it('should work with simple insert', async () => {
  let { sequelize, User } = define();

  let inserted = await User.create({
    name: 'Michiel Dral',
    email: 'm.c.dral@gmail.com',
    age: 22,
    verified: true,
  });

  expect(inserted).toMatchSnapshot();
});
