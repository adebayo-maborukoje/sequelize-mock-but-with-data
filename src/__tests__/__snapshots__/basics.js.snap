// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should work with simple insert 1`] = `
Object {
  "__table_name__": "User",
  "age": 22,
  "createdAt": 2019-01-16T17:34:47.828Z,
  "email": "m.c.dral@gmail.com",
  "name": "Michiel Dral",
  "updatedAt": 2019-01-16T17:34:47.828Z,
  "verified": true,
}
`;
